clinica/iotools/converters/habs_to_bids/habs_to_bids.py

```diff
@@ -0,0 +1,317 @@
+from enum import Enum
+from io import TextIOBase
+from pathlib import Path
+from typing import Dict, List, Optional, Tuple
+
+import pandas as pd
+from pandas import DataFrame, Series
+from pydantic import BaseModel
+
+PROTOCOL_TO_BIDS = {
+    "3DFLAIR": {"datatype": "anat", "modality": "FLAIR"},
+    "ADNI_1X-MPRAGE": {"datatype": "anat", "modality": "T1w"},
+    "ADNI_2X-MPRAGE": {"datatype": "anat", "modality": "T1w"},
+    "FDG": {
+        "datatype": "pet",
+        "modality": "pet",
+        "trc_label": "18FFDG",
+    },
+    "FTP_75-105": {
+        "datatype": "pet",
+        "modality": "pet",
+        "trc_label": "18FFTP",
+    },
+    "FTP_80-100": {
+        "datatype": "pet",
+        "modality": "pet",
+        "trc_label": "18FFTP",
+    },
+    "PIB_40-60": {
+        "datatype": "pet",
+        "modality": "pet",
+        "trc_label": "11CPIB",
+    },
+}
+
+
+class BidsDatasetDescription(BaseModel):
+    class DatasetType(str, Enum):
```

Comment:
Could you please add a small docstring for this class? It may be useful for other converters... isn't?